doc/sphinx/proof-engine/tactics.rst

```diff
@@ -157,10 +157,10 @@ The :n:`eqn:` construct in various tactics uses :n:`@naming_intropattern`.
 
 Use these elementary patterns to specify a name:
 
-* :n:`@ident` - use the specified name
-* :n:`?` - let Coq choose a name
-* :n:`?@ident` - generate a name that begins with :n:`@ident`
-* :n:`_` - discard the matched part (unless it is required for another
+* :n:`@ident` — use the specified name
+* :n:`?` — let Coq choose a name
+* :n:`?@ident` — generate a name that begins with :n:`@ident`
+* :n:`_` — discard the matched part (unless it is required for another
```

Comment:
@Zimmi48, is there a guideline for which dash to use?  I find 129 uses of " - " in *.rst, some of which should change if this is the preferred formatting--not including the changes in this PR.

Reply:
I recently read that m-dash should be used as separators, n-dash only to mark a range, and hyphens only for composed words. But because hyphens are easier accessed on a standard keyboard, they are much abused.

Reply:
These look like n-dashes rather than m-dashes to me, but it already seems like an improvement anyway.

Reply:
> look like n-dashes rather than m-dashes to me

They are em-dashes ;-)

<img width="888" alt="Screen Shot 2019-10-22 at 17 54 00GMT+4" src="https://user-images.githubusercontent.com/870172/67293095-ffa96580-f4f4-11e9-98ff-e9da8a6ad77f.png">